src/components/Media/index.js

```diff
@@ -72,6 +72,7 @@ export default class Media extends PureComponent {
 
   renderIcon(props) {
     const {icon, icons, iconColor: fill, iconSize: size, theme} = props
+    if (!icons) return null
```

Comment:
isn't icons required by PropTypes?

Reply:
Not currently. Do you think it should be required?

line 43 of `src/components/Media/index.js`:
```js
/** display icon */
icon: PropTypes.oneOf([load, loading, loaded, error, noicon, offline]),
```

I added that check because by not providing icon as a prop, the render threw.

Reply:
Yes it is required. It is not provided here to be shakable, so if you want to provide your icons you can do this and you will not need to have default icons in the bundle. But if you want to use default ones there is a component with defaults preseted (IdealImageWithDefaults)

Reply:
Okay, i will fix that then! Should I also make icon required in the PropTypes?

Reply:
Yes icons, icon and theme are all required. I pushed change to the master, you can rebase